app/utils/hooks/useToggle.js

```diff
@@ -0,0 +1,13 @@
+import { useState } from 'react';
+
+/**
+ * Returns a stateful boolean value and one function to toggle it
+ *
+ * @param initialValue
+ * @returns {(boolean|(function(): void))[]}
```

Comment:
This is a bit more precise type (also a correct TypeScript type :smile: ) that works well in VSCode:
```suggestion
 * @returns {[boolean, () => void]}
```

I'm not sure if WebStorm parses it correctly too

Reply:
![image](https://user-images.githubusercontent.com/5202105/106464687-44ae9c80-6499-11eb-88eb-16154ee6f7c7.png)

I'm not sure how it should be for WebStorm, but neither the original part (taken from `useBoolean`) nor the one you suggested is visible in the popup provided by the WebStorm. Don't know if there is any other way to trigger it.

Anyways, I can change it according to your suggestion if that is the valid way (similiarly I can change `useBoolean`). @kubama do you see it differently?

Reply:
Works fine in WebStorm. Changed.

Still there are other places using that syntax - `useResettableState`, `useBoolean`, `useErrors`. That can be handled in a separate issue, I think.

Reply:
Yup, definitely :smile: We can refactor it as we go :slightly_smiling_face: